src/zenml/integrations/seldon/seldon_client.py

```diff
@@ -476,7 +476,9 @@ def _initialize_k8s_clients(self) -> None:
     def sanitize_labels(labels: Dict[str, str]) -> None:
         """Update the label values to be valid Kubernetes labels"""
         for key, value in labels.items():
-            labels[key] = re.sub(r"[^0-9a-zA-Z-_\.]+", "_", value)
+            # Kubernetes labels must be alphanumeric and no longer than
+            # 63 characters
+            labels[key] = re.sub(r"[^0-9a-zA-Z-_\.]+", "_", value)[:63]
```

Comment:
is it possible that in some cases the final value could be ending with "_" after cutting it to 63 which may cause problems since the name should be starting and ending with alphanumeric?

Reply:
Good point. Done.

Reply:
Looks Great!